scripts/test

```diff
@@ -2,4 +2,5 @@
 
 set -e
 
-pytest --cov-report term --cov-report xml --cov=lambdas --cov=datapump_utils tests/
+pytest --cov-report term --cov-report xml --cov=lambdas --cov=datapump_utils tests/test_environment_tests/
+pytest --cov-report term --cov-report xml --cov=lambdas --cov=datapump_utils tests/mocked_environment_tests/
```

Comment:
why not all tests?

Reply:
The mock environment ended up interfering with the tests using our actual -test buckets, since I had to set all the AWS credentials to dummy values. So I just separated into two pytest calls so they'd run in different shells

Reply:
Still in alpha, but this might help: https://pytest-ordering.readthedocs.io/en/develop/